src/Paket.Core/PlatformDetection.fs

```diff
@@ -8,27 +8,37 @@ let isMonoRuntime =
     not (Object.ReferenceEquals(Type.GetType "Mono.Runtime", null))
 
 /// Determines if the current system is an Unix system
-let isUnix = Environment.OSVersion.Platform = PlatformID.Unix
+let isUnix = Paket.Utils.isUnix // Environment.OSVersion.Platform = PlatformID.Unix
```

Comment:
I wanted to minimize the diff here. Removing this would increase the diff. Note that this was duplicate code... Should I still remove? Or do you mean the comments?